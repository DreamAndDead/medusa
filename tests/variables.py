a = 45
b = 100

a, b = b, a

c = a * b
d = c / (1 + 1)

print(a)
print(b)
print(c)
print(d)