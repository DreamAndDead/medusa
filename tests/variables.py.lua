dofile('./pythonlua/luainit.lua')

local a = 45
local b = 100
local a, b = b, a
local c = (a * b)
local d = (c / (1 + 1))
print(a)
print(b)
print(c)
print(d)