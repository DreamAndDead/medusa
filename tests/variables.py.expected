100
45
4500
2250
